apiary.apib

```diff
@@ -3701,6 +3701,14 @@ Get a list of tracked time.
                         + milestone
                         + ticket
                 + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
+            + subject_types (array, optional) - Include all tracked time with one of the provided subject types. For tracked time without a subject type, provide `null`
+                + Members
+                    + company
+                    + contact
+                    + event
+                    + todo
```

Comment:
Q: Isn't `todo` a deprecated term? Shouldn't this be `task`?

Reply:
In our API, it's still `todo`. We should change that to task, but that's very much a breaking change, witch requires a new version, and is out of scope for the moment.

Reply:
🤔 I seem to recall some endpoints (as well as sideloader?) moving to `task` whenever possible while still supporting legacy `todo` when required..
Will try to find some examples.